src/phoenix/exceptions.py

```diff
@@ -1,2 +1,6 @@
 class PhoenixException(Exception):
     pass
+
+
+class PhoenixContextLimitExceeded(PhoenixException):
+    pass
```

Comment:
nit: I kinda feel this should be co-located with models, not at the root exeptions